source/faqs.rst

```diff
@@ -80,7 +80,7 @@ What versions are supported?
 Operating Systems
 ~~~~~~~~~~~~~~~~~
 
-Bioconda only supports 64-bit Linux and macOS. ARM is not currently supported.
+Bioconda only supports Linux (64-bit and AArch64) and macOS (64-bit). ARM is not currently supported for macOS.
```

Comment:
`aarch64` is also 64 bit. Maybe `64-bit` should be changed to `x86_64` or `AMD64` ?!

Reply:
Good point!